docs/business-apps/power-apps/get-started/create-your-first-custom-form.md

```diff
@@ -42,7 +42,9 @@ Before starting, make sure your SharePoint site is set up with the [required lis
     
     1. When you select a control on the canvas, on the right-hand panel, the **Properties** pane associated with the control appears.
 
-    ![Image control with Properties panel](../../../images/gs01-image-control-selected.png)
+    ![Image control with Properties panel](../../../images/gs01-image-control-selected.png)<img width="403" alt="Lists_Integrate_PowerApps_CreateApp" src="https://user-images.githubusercontent.com/70417807/117352154-7bfb6380-ae63-11eb-9fc7-f275a62715f2.png">
```

Comment:
Images should be added seperately as files to the folder structure, then reference them in the markdown. Can you please address before we can merge your submission? thanks!

Reply:
I think I figured it out once I was looking in the MattEEvans:patch-1 branch. I submitted the update.

Reply:
I discovered that my earlier attempt to upload a file as an attachment (the other day) had created some changes I didn't intend to make, so I backed those changes out. Hopefully we're good now.

Reply:
@MattEEvans  please see the build errors/warnings in the last automated check. The image can't be found because it's in the wrong folder. It should be in the **./docs/images/** folder, while it appears you stuck it in the root **./images** folder. Can you please move the image in your PR to the correct location next to the other images in this article?

Also notice the build warning for the duped image `alt` tags. While you may not have been the author, we can't merge this with that error, so if you can update one of the alt tags to be more descriptive, we can process this further.